content/800-data-platform/050-data-proxy/index.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Data Proxy'
+metaTitle: 'Data Proxy'
+metaDescription: 'Data Proxy in the Prisma Data Platform provides database connection management and pooling, load balancing, scaling, and fault tolerance features so that you can manage and scale database connections for applications that use Prisma ORM. With Data Proxy, your application can scale and maintain predictable database performance as it also limits the total number of database connections.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Serverless functions are ephemeral and short-lived, so their database connections are numerous and brief.
+
+Because of this, using traditional databases in serverless environments often leads to the exhaustion of the maximum number of concurrent database connections and increased latencies when each request needs to establish a database connection.
+
+Data Proxy in the Prisma Data Platform provides database connection management and pooling, load balancing, scaling, and fault tolerance so that you can manage and scale database connections for applications that run with Prisma ORM in serverless environments. With Data Proxy, your application can scale and maintain predictable database performance as it also limits the total number of database connections.
```

Comment:
```suggestion
Data Proxy in the Prisma Data Platform provides database connection management and pooling, load balancing, scaling, and fault tolerance so that you can manage and scale database connections for applications that run with Prisma ORM in serverless environments. With Data Proxy, your application can scale and maintain predictable database performance because it also limits the total number of database connections.
```